s3/s3.go

```diff
@@ -192,7 +193,9 @@ func NewS3Client(ctx context.Context, opts S3ClientOpts) (S3Client, error) {
 	if opts.Trace {
 		minioClient.TraceOn(log.StandardLogger().Out)
 	}
-
+	if opts.DisableDualStack {
```

Comment:
```suggestion
	if !opts.EnableDualstack {
```

I would match the existing naming and capitalization of Minio's

Reply:
I did it intentionally since Minio enables it by default and if the client doesn't specify `true` in this field explicitly, it always disable it. But, if you think it's fine, I follow your suggestion